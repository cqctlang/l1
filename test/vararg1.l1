printlist = @lambda(l) {
	@local i;
	for(i = 0; i < length(l); i++)
		print(listref(l, i));
};

fn = @lambda(arg ...) {
	@local x;
	x = arg;
	print(length(x));
	printlist(x);
};

fn(1,2,3,4,5);
