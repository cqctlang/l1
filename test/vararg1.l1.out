entry: var params, 1 locals, 0 temps
L5: 1 params, 1 locals, 0 temps
L18: var params, 2 locals, 0 temps
5
1
2
3
4
5
